docs/framework/wpf/advanced/properties-wpf.md

```diff
@@ -15,10 +15,10 @@ ms.assetid: d6e0197f-f2c4-48ed-b45b-b9cdb64aab1c
   
 ## In This Section  
  [Dependency Properties Overview](../../../../docs/framework/wpf/advanced/dependency-properties-overview.md)  
- [Attached Properties Overview](../../../../docs/framework/wpf/advanced/attached-properties-overview.md)  
- [Dependency Property Callbacks and Validation](../../../../docs/framework/wpf/advanced/dependency-property-callbacks-and-validation.md)  
+ [Attached Properties Overview](../../../../docs/framework/wpf/advanced/attached-properties-overview.md)   
  [Custom Dependency Properties](../../../../docs/framework/wpf/advanced/custom-dependency-properties.md)  
- [Dependency Property Metadata](../../../../docs/framework/wpf/advanced/dependency-property-metadata.md)  
+ [Dependency Property Metadata](../../../../docs/framework/wpf/advanced/dependency-property-metadata.md)
```

Comment:
the formatting is slightly broken now because the two spaces at the end were removed. You could add them back or make this list a bulleted list instead.

![image](https://user-images.githubusercontent.com/12971179/53230321-4af43700-363b-11e9-8110-7631a483001c.png)

Reply:
@mairaw thanks for your review, I made changes